src/Compilation.zig

```diff
@@ -2828,7 +2870,7 @@ fn updateStage1Module(comp: *Compilation, main_progress_node: *std.Progress.Node
     man.hash.add(comp.bin_file.options.function_sections);
     man.hash.add(comp.bin_file.options.is_test);
     man.hash.add(comp.bin_file.options.emit != null);
-    man.hash.addOptionalEmitLoc(comp.emit_h);
+    man.hash.add(comp.c_header != null);
```

Comment:
There should be a `addOptionalEmitLoc(header.emit_loc.?)` here if `c_header != null`.

Reply:
e5fb9d0